lib/src/main/kotlin/at/bitfire/synctools/icalendar/validation/FixInvalidDayOffsetPreprocessor.kt

```diff
@@ -6,13 +6,16 @@
 
 package at.bitfire.synctools.icalendar.validation
 
+import androidx.annotation.VisibleForTesting
+
 /**
  * Fixes durations with day offsets with the 'T' prefix.
  * See also https://github.com/bitfireAT/ical4android/issues/77
  */
-class FixInvalidDayOffsetPreprocessor : StreamPreprocessor() {
+class FixInvalidDayOffsetPreprocessor : StreamPreprocessor {
 
-    override fun regexpForProblem() = Regex(
+    @VisibleForTesting
+    val regexpForProblem = Regex(
```

Comment:
Would it be enough to use `internal`? I usually combine `@VisibleForTesting` with `internal` so that tests can access the field, but other packages can't.